l10n_th_tax_report/reports/tax_report_xlsx.py

```diff
@@ -0,0 +1,150 @@
+                "data": {"value": self._render("row_pos")},
+                "width": 3,
+            },
+            "2_tax_date": {
+                "header": {"value": "Date"},
+                "data": {"value": self._render("tax_date")},
+                "width": 12,
+            },
+            "3_tax_invoice": {
+                "header": {"value": "Number"},
+                "data": {"value": self._render("tax_invoice_number")},
+                "width": 18,
+            },
+            "4_partner_name": {
+                "header": {"value": "Cust./Sup."},
+                "data": {"value": self._render("partner_name")},
+                "width": 30,
+            },
+            "5_partner_vat": {
+                "header": {"value": "Tax ID"},
+                "data": {"value": self._render("partner_vat")},
+                "width": 15,
+            },
+            "6_partner_branch": {
+                "header": {"value": "Branch ID"},
+                "data": {"value": self._render("partner_branch")},
+                "width": 12,
+            },
+            "7_tax_base_amount": {
+                "header": {"value": "Base Amount"},
+                "data": {
+                    "value": self._render("tax_base_amount"),
+                    "format": self.format_tcell_amount_right,
+                },
+                "width": 21,
+            },
+            "8_tax_amount": {
+                "header": {"value": "Tax Amount"},
+                "data": {
+                    "value": self._render("tax_amount"),
+                    "format": self.format_tcell_amount_right,
+                },
+                "width": 21,
+            },
+            "9_doc_ref": {
+                "header": {"value": "Doc Ref."},
+                "data": {"value": self._render("doc_ref")},
+                "width": 18,
+            },
+        }
+        ws_params = {
+            "ws_name": "TAX Report",
+            "generate_ws_method": "_vat_report",
+            "title": "TAX Report",
+            "wanted_list": [k for k in sorted(tax_template.keys())],
+            "col_specs": tax_template,
+        }
+        if objects.tax_id.type_tax_use == "sale":
+            ws_params["ws_name"] = "Sale TAX Report"
+            ws_params["title"] = "Sale TAX Report"
+        elif objects.tax_id.type_tax_use == "purchase":
+            ws_params["ws_name"] = "Purchase TAX Report"
+            ws_params["title"] = "Purchase TAX Report"
+
+        return [ws_params]
+
+    def _vat_report(self, wb, ws, ws_params, data, objects):
+        ws.set_portrait()
+        ws.fit_to_pages(1, 0)
+        ws.set_header(self.xls_headers["standard"])
+        ws.set_footer(self.xls_footers["standard"])
+        self._set_column_width(ws, ws_params)
+        row_pos = 0
+        # title
+        row_pos = self._write_ws_title(ws, row_pos, ws_params, True)
+        # company data
+        ws.write_column(row_pos, 1, ["Period :", "Partner :"], self.format_left_bold)
```

Comment:
```suggestion
        ws.write_column(row_pos, 1, ["Period :", "Partner :"], FORMATS["format_left_bold"])
```

change all `self.format_xxxx` to `FORMATS["format_xxxx"]`